src/Gateio/Utils.jl

```diff
@@ -0,0 +1,21 @@
+# Gateio/Utils
+
+function Serde.deser(::Type{<:GateioData}, ::Type{<:Maybe{NanoDate}}, x::Int64)::NanoDate
+    return unixnanos2nanodate(x * 1e9)
+end
+
+function Serde.deser(::Type{<:GateioData}, ::Type{<:Maybe{NanoDate}}, x::AbstractString)::NanoDate
+    return unixnanos2nanodate(parse(Int64, x) * 1e9)
+end
+
+function Serde.SerQuery.ser_value(::Type{<:GateioCommonQuery}, ::Val{:signTimestamp}, v::Any)::Nothing
+    return nothing
+end
+
+function Serde.SerQuery.ser_value(::Type{<:GateioCommonQuery}, ::Val{:sign}, v::Any)::Nothing
+    return nothing
+end
```

Comment:
There's a  `ignore_field(::Type{T}, ::Val{x})::Bool where {T,x} = false` method in Serde defined for specifically for that case.

Reply:
`ser_ignore_field`